consts/validation.js

```diff
@@ -16,7 +16,8 @@ const enums = {
   SPOKEN_LANG_ENUM: ['English', 'Ukrainian', 'Polish', 'German', 'French', 'Spanish', 'Arabic'],
   SUBJECT_LEVEL_ENUM: ['Beginner', 'Intermediate', 'Advanced', 'Test Preparation', 'Professional'],
   ROLE_ENUM: ['student', 'tutor', 'admin'],
-  STATUS_ENUM: ['active', 'blocked']
+  STATUS_ENUM: ['active', 'blocked'],
+  COOPERATION_STATUS: ['active', 'closed', 'pending']
```

Comment:
it makes sense to change the order of those, place it in a logical order
pending, active, closed

and then do not forget to change index for default option inside the model